The test is failing with:
```
Caused by: java.util.NoSuchElementException: SRCFG00027: Could not find a mapping for io.quarkus.arc.test.config.ConfigPropertiesTest$ServerConfigProperties
	at io.smallrye.config.ConfigMappings.getConfigMapping(ConfigMappings.java:73)
	at io.smallrye.config.SmallRyeConfig.getConfigMapping(SmallRyeConfig.java:398)
	at io.quarkus.arc.runtime.ConfigMappingCreator.create(ConfigMappingCreator.java:28)
	at io.quarkus.arc.test.config.ConfigPropertiesTest_ServerConfigProperties_b3cdb56c503cd9264db6a7e03a065e674049c2c1_Synthetic_Bean.create(ConfigPropertiesTest_ServerConfigProperties_b3cdb56c503cd9264db6a7e03a065e674049c2c1_Synthetic_Bean.zig:166)
	at io.quarkus.arc.test.config.ConfigPropertiesTest_ServerConfigProperties_b3cdb56c503cd9264db6a7e03a065e674049c2c1_Synthetic_Bean.get(ConfigPropertiesTest_ServerConfigProperties_b3cdb56c503cd9264db6a7e03a065e674049c2c1_Synthetic_Bean.zig:197)
	at io.quarkus.arc.test.config.ConfigPropertiesTest_ServerConfigProperties_b3cdb56c503cd9264db6a7e03a065e674049c2c1_Synthetic_Bean.get(ConfigPropertiesTest_ServerConfigProperties_b3cdb56c503cd9264db6a7e03a065e674049c2c1_Synthetic_Bean.zig:220)
	at io.quarkus.arc.impl.CurrentInjectionPointProvider.get(CurrentInjectionPointProvider.java:52)
	at io.quarkus.arc.test.config.ConfigPropertiesTest_Bean.create(ConfigPropertiesTest_Bean.zig:711)
	... 51 more
```
which might or might not be related given we have some random failures in this test for a while.

We really need to focus on fixing these tests...

--- a/bom/application/pom.xml
+++ b/bom/application/pom.xml
@@ -42,7 +42,7 @@
         <microprofile-jwt.version>1.2</microprofile-jwt.version>
         <microprofile-lra.version>1.0</microprofile-lra.version>
         <smallrye-common.version>1.6.0</smallrye-common.version>
-        <smallrye-config.version>2.4.3</smallrye-config.version>
+        <smallrye-config.version>2.4.4</smallrye-config.version>
         <smallrye-health.version>3.1.1</smallrye-health.version>
         <smallrye-metrics.version>3.0.1</smallrye-metrics.version>
         <smallrye-open-api.version>2.1.10</smallrye-open-api.version>

